server/routes/activity-list/activityListRoutes.ts

```diff
@@ -1,18 +1,25 @@
 import { RequestHandler, Router } from 'express'
 import asyncMiddleware from '../../middleware/asyncMiddleware'
-import ActivityListRouteHandler from './handlers/activityListRouteHandler'
+import ActivityListRouteHandler from './handlers/ActivityListRouteHandler'
 import SelectActivityLocationRouteHandler from './handlers/SelectActivityLocationRouteHandler'
 import { Services } from '../../services'
+import fetchActivityList from '../../middleware/fetchActivityList'
 
 export default ({ prisonService }: Services): Router => {
   const router = Router()
-  const get = (path: string, handler: RequestHandler) => router.get(path, asyncMiddleware(handler))
+  const getSelectActivityLocation = (path: string, handler: RequestHandler) =>
+    router.get(path, asyncMiddleware(handler))
+  const postSelectActivityLocation = (path: string, handler: RequestHandler) =>
```

Comment:
this can just be called `post` and `get`?

Reply:
There are two 'gets' in there atm and i think ill be adding at least one more post so i wanted to differentiate

Reply:
But all of the `post` methods do the same thing?

Reply:
theres only one post method atm. Plan is to add another - not sure if i will be able to use same method yet - if so i will rename it to post

Reply:
Even the 2 get methods `getActivityList` and `getSelectActivityLocation` are identical. It can just be called `get`